stock_by_warehouse/static/src/xml/template.xml

```diff
@@ -30,4 +30,37 @@
             </div>
         </div>
     </t>
+
+    <t t-name="StockAvailabilityAlert">
+        <p class="availability_product my-0 mr-2">
+            The product is available in: <t t-raw="locations_available"/> locations.
+            <a role="button" tabindex="0" class="js_available_info fa fa-bars"></a>
+        </p>
+    </t>
+
+    <t t-name="StockAvailabilityPopOver">
+        <div>
+            <table class="table table-striped table-bordered">
+                <thead>
+                    <tr>
+                        <th title="Stock Location">Location</th>
+                        <th title="Quantity Available">Stock Quantity</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <t t-foreach="lines" t-as="warehouse">
+                        <tr class="table-info">
+                            <td>Warehouse: <t t-raw="warehouse.name"/></td>
```

Comment:
Is `t-raw` really required? (i.e. it contains HTML code)

If not, `t-esc` is preferred

Reply:
Done